src/python/grpcio_testing/grpc_testing/__init__.py

```diff
@@ -14,9 +14,9 @@
 """Objects for use in testing gRPC Python-using application code."""
 
 import abc
+import six
```

Comment:
This is a weird change. That's pylint's complaint about this import order?

Reply:
It complained about `six` being after `from google.protobuf...`. I suspect "common library" imports should appear earlier than other local ones, perhaps? This was the only warning of this category so I just fixed it without reading too much into it.